string req
---
float32[] data
